src/lib/docs/SessionKit/LoginResult.md

```diff
@@ -1,19 +1,18 @@
 ---
 title: LoginResult
-description: The data returned from a call to the login method from the Session Kit
-category: SessionKit
+description: The data returned from a call to the login method from the Session Kit.
+category: Session Kit
 published: true
 hidden: true
-requiresReview: true
 ---
 
 # LoginResult
 
-Upon a successful call to the [Login](/docs/sessionkit/login) method of the [SessionKit](/docs/sessionkit/session-kit-factory), the response returned will be an object that matches the `LoginResult` interface.
+Upon a successful call to the [Login](/docs/sessionkit/login) method of the [Session Kit](/docs/sessionkit/session-kit-factory), the response returned will be an object that matches the `LoginResult` interface.
```

Comment:
I've been intentionally doing 2 things here: 

- Using `SessionKit` when referring to the "code object" (you can see the url goes to the "session-kit-factory" page)
- Using `Session Kit` when it refers to the overall product (and the url goes to the landing page of /docs/sessionkit) 

A lot of these instances that were changed in this PR break that pattern and before we merge any of this in, I want to just make a decision if we're changing that everywhere.